src/components/InfoPanel/InfoPanel.js

```diff
@@ -42,4 +36,20 @@ const InfoPanel = () => {
   );
 };
 
+InfoPanel.defaultProps = {
```

Comment:
Instead of passing props in I think we'll want to stick with the `redux` data model here. That is, fetching the necessary data from a selector. This will give us the ability to modify the data displayed in this component without worrying about its parent. I can help you out doing this if you'd like.